src/react/components/shell/sidebar.tsx

```diff
@@ -39,6 +39,13 @@ export function Sidebar({ project }) {
                         <FontIcon iconName="MachineLearning" />
                     </ConditionalNavLink>
                 </li>
+                {/* <li>
```

Comment:
remove dead code before check into master

Reply:
it is not dead code, comment this out just because we need to hide the entrance to model compose page. Do we really should also remove this one right now?